sdk/identity/azure-identity/README.md

```diff
@@ -300,10 +301,76 @@ public void createAuthCodeCredential() {
 }
 ```
 
+### Authenticating a user account with Azure CLI
+This example demonstrates authenticating the `KeyClient` from the [azure-security-keyvault-keys][keys_client_library] client library using the `AzureCliCredential` on a workstation with Azure CLI installed and signed in.
+
+See more about how to configure Azure CLI in [Sign in Azure CLI for AzureCliCredential](#sign-in-azure-cli-for-azureclicredential).
+
+<!-- embedme ../../keyvault/azure-security-keyvault-secrets/src/samples/java/com/azure/security/keyvault/secrets/IdentityReadmeSamples.java#L148-L159 -->
+```java
+/**
+ * Authenticate with Azure CLI.
+ */
+public void createAzureCliCredential() {
+    AzureCliCredential cliCredential = new AzureCliCredentialBuilder().build();
+
+    // Azure SDK client builders accept the credential as a parameter
+    SecretClient client = new SecretClientBuilder()
+        .vaultUrl("https://{YOUR_VAULT_NAME}.vault.azure.net")
+        .credential(cliCredential)
+        .buildClient();
+}
+```
+
+### Authenticating a user account with IntelliJ IDEA
+This example demonstrates authenticating the `KeyClient` from the [azure-security-keyvault-keys][keys_client_library] client library using the `IntelliJCredential` on a workstation with IntelliJ IDEA installed, and the user has signed in with an Azure account.
+
+See more about how to configure your IntelliJ IDEA in [Sign in Azure Toolkit for IntelliJ for IntelliJCredential](#sign-in-azure-toolkit-for-intellij-for-intellijcredential).
+
+<!-- embedme ../../keyvault/azure-security-keyvault-secrets/src/samples/java/com/azure/security/keyvault/secrets/IdentityReadmeSamples.java#L174-L188 -->
+```java
+/**
+ * Authenticate with IntelliJ IDEA.
+ */
+public void createIntelliJCredential() {
+    IntelliJCredential intelliJCredential = new IntelliJCredentialBuilder()
+        // KeePass configuration required for Windows
```

Comment:
Maybe elaborate the comment here ?
required only for Windows.
No configuration needed for Linux / Mac

Reply:
modified